templates/iot/examples/k8s-tls/create

```diff
@@ -36,8 +37,13 @@ echo $CA_SUBJECT
 echo $PK
 
 function createServiceKeyAndCert() {
-  openssl req -config "$SCRIPTDIR/ca.cnf" -nodes -newkey rsa:4096 -keyout "$OUTDIR/$1-key.pem" -days 365 -subj "/O=EnMasse/OU=IoT/CN=$1" | \
-  SAN="DNS.1:${1}.svc,DNS.2:${1}.${NAMESPACE}.svc,DNS.3:localhost" openssl x509 -req -extfile "$SCRIPTDIR/ca.cnf" -extensions "san_ext" -out "$OUTDIR/$1.pem" -days 365 -CA "$OUTDIR/ca-cert.pem" -CAkey "$OUTDIR/ca-key.pem" -CAcreateserial
+  DNSHOSTNAME=${CLUSTER}
+  if [ ${CLUSTER} != "localhost" ]; then
+    DNSHOSTNAME=${1}-${NAMESPACE}.${CLUSTER}
+  fi
+  echo $DNSHOSTNAME
+  openssl req -config "$SCRIPTDIR/ca.cnf" -nodes -newkey rsa:4096 -keyout "$OUTDIR/$1-key.pem" -days 365 -subj "/O=EnMasse/OU=IoT/CN=$DNSHOSTNAME" | \
+  SAN="DNS.1:${1}.svc,DNS.2:${1}.${NAMESPACE}.svc,DNS.3:$DNSHOSTNAME" openssl x509 -req -extfile "$SCRIPTDIR/ca.cnf" -extensions "san_ext" -out "$OUTDIR/$1.pem" -days 365 -CA "$OUTDIR/ca-cert.pem" -CAkey "$OUTDIR/ca-key.pem" -CAcreateserial
```

Comment:
I think we should leave the CN and DNS entry for localhost in place. Simply add another `DNS.4` entry with `$DNSHOSTNAME`

Reply:
@ctron should be fixed now